cluster/ubuntu/config-default.sh

```diff
@@ -37,6 +37,10 @@ export FLANNEL_NET=${FLANNEL_NET:-172.16.0.0/16}
 # Admission Controllers to invoke prior to persisting objects in cluster
 export ADMISSION_CONTROL=NamespaceLifecycle,LimitRanger,ServiceAccount,ResourceQuota,SecurityContextDeny
 
+# Path to the config file or directory of files of kubelet
+export KUBELET_CONFIG=${KUBELET_CONFIG:-""}
```

Comment:
I'm not sure you ever want to pass `--config=` to the kubelet. If you want to default to not passing a config flag you should check to see if this var is set and if so pass the flag with the value and otherwise omit the flag. If you always want to pass the flag but want to allow it to be overridden, then you could make the default value be `/etc/kubernetes/manifests` to match your previous PR.